spec/services/articles/destroyer_spec.rb

```diff
@@ -15,9 +15,9 @@
 
   it "schedules removing notifications if there are comments" do
     create(:comment, commentable: article)
-    expect do
+    sidekiq_assert_enqueued_jobs(2, only: Notifications::RemoveAllWorker) do
```

Comment:
here we went from an assertion that checks for 1 job, to one that checks for 2 jobs, what changed?

Reply:
I thought should be running two jobs in this case. But I re-analyzed the code and I made a mistake changing the numbers of jobs, in both specs.